src/property-filter/filtering-token/styles.scss

```diff
@@ -51,6 +57,13 @@ $inner-token-padding-inline: styles.$control-padding-horizontal;
   }
 }
 
+.inner-token {
+  border-start-start-radius: $border-radius-inner-token;
+  border-start-end-radius: $border-radius-inner-token;
+  border-end-start-radius: $border-radius-inner-token;
+  border-end-end-radius: $border-radius-inner-token;
```

Comment:
Why do we do it this way and not just use `border-radius: $border-radius-inner-token;`?

Reply:
The border-radius is banned by stylelint because of RTL.